less/colors.less

```diff
@@ -172,8 +172,9 @@
 
 	@obj-key: #0068a1;
 	// @const: @darker;
-	@const: #88428F;
-	@const: #8F4242;
+	// @const: #88428F;
+	// @const: #8F4242;
+	// @const: var(--fourth-color);
 
```

Comment:
Why are these commented out lines there?

Reply:
Would be nice to drop the commented stuff, they are in the git history so can always be recovered.

Reply:
Well. I believe two of them were already there. I left my var in there. I'll clean that up.

Reply:
Actually they were all there, but uncommented. I must have been trying to overwrite them. I don't understand why they were all there overwriting each other.

Reply:
OK.